app/src/main/res/values/donottranslate.xml

```diff
@@ -27,4 +27,8 @@
 
     <!-- We won't have a translation for this, to be used later. -->
     <string name="trackersOverviewEmpty">No tracking requests were blocked from loading on this page. If a company\'s requests are loaded, it can allow them to profile you.</string>
+
+    <!-- Notify Me View on Downloads-->
+    <string name="downloadsNotifyMeTitle" translatable="false">Find out when downloads are ready</string>
```

Comment:
nit: move this to `common-ui`?

Reply:
Same as above.
_I don't think this string belongs to the common-ui. The common-ui module should not know how the component is used in other modules._

Reply:
ditto